common/ellipses.js

```diff
@@ -136,10 +140,12 @@
                                                     scope.$emit('record-modified');
                                                 }).catch(function(error) {
                                                     scope.$emit('error', response);
+                                                    // TODO: Throw the error instead
                                                     ErrorService.catchAll(error);
                                                 });
                                             } else {
                                                 scope.$emit('error', response);
+                                                // TODO: Throw the error instead
                                                 ErrorService.catchAll(error);
                                             }
                                         }).catch(function (error) {
```

Comment:
10 lines below this, [delete](https://github.com/informatics-isi-edu/chaise/blob/rec-errors/common/ellipses.js#L165) should also be wrapped.